src/resources/elements/primeDesignSystem/_variables.scss

```diff
@@ -9,7 +9,7 @@ $spacing-small: 5px;
 $spacing-medium-small: 7.5px;
 $spacing-normal: 10px;
 
-$input-height: 40px;
+$input-height: 46px;
```

Comment:
did you pass this by gilad?  I believe he put this in there to support the pinput-select

Reply:
I was wondering too, but apparently, the size has changed on Figma.

Reply:
Yes, I took that from the Figma